beast/observationmodel/observations.py

```diff
@@ -125,35 +99,33 @@ def setFilters(self, filters):
 
     def getMags(self, num, filters):
         raise Exception('Do not use as magnitudes')
-        return np.array([ self.data[tt][num] - self.distanceModulus for tt in filters])
```

Comment:
Hopefully this comment was not too long-winded.  Just trying to clearly state the issue.  Not sure I did.  Having some allergy fuzziness issues today.